### ISSUE TITLE
Applying style token stopped working in v8 versions. In 7.9.5 version it works.

### ISSUE THREAD

**Issue by Juraj2** [notepad-plus-plus/notepad-plus-plus#10337]
### Description of the Issue
Applying style token stopped working in v8 versions 32bit. In v7.9.5 32bit it works. In 8.x it does not. Last veriffied on v 8.1.4 that it still doesn't work.  

### Steps to Reproduce the Issue
1.  Select multiple words in the text
2.  Right click on the selection (opens context menu)
3.  Choose "Style Token" option and then "Using 1st Style" option
![image](https://user-images.githubusercontent.com/39399251/128316419-3342054e-6c95-4101-bc2f-a3a2e1317038.png)

### Expected Behavior
The selected text should be colored with style 1.
![image](https://user-images.githubusercontent.com/39399251/128316511-ac1a4ef4-75b7-4075-bd42-ea5d93d93012.png)

### Actual Behavior
The selected text is not colored. Nothing happens.
![image](https://user-images.githubusercontent.com/39399251/128316597-9072a3db-bfa5-48ab-a09c-fed85581b172.png)

**Comment by mere-human** [comment]
Try changing these options
![image](https://user-images.githubusercontent.com/9664141/128317939-d82e696d-e060-417d-979f-d4c0ecffa69d.png)

**Comment by stgmviknesh** [comment]
The above-suggested method didn't fix the issue. After highlighting
![image](https://user-images.githubusercontent.com/57669079/128396325-5e79035d-eb7b-4bdd-a753-e80bff373410.png)

it doesn't highlight like the previous versions
![image](https://user-images.githubusercontent.com/57669079/128396497-51df945a-6272-4e5e-ab4b-f65fa8321175.png)

**Comment by mere-human** [comment]
@stgmviknesh 
So I tried disabling _Preferences > Highlighting > Mark All > Match whole word only_
And it looks this way:
![image](https://user-images.githubusercontent.com/9664141/128397936-f99ad711-244a-4b28-90dd-b33c923ed796.png)

Can you try the same?

```
Notepad++ v8.1.2   (64-bit)
Build time : Aug  5 2021 - 20:47:30
Path : ...\Notepad++.exe
Command Line : -multiInst -nosession
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19043.1110
Current ANSI codepage : 1252
Plugins : none

```

**Comment by ArkadiuszMichalski** [comment]
@stgmviknesh 
In context menu you have only "one mark", for more you must use Search menu.
![image](https://user-images.githubusercontent.com/2730894/128403042-870d2cb3-f3df-4078-9b96-4987ac95df7e.png)

**Comment by meiko-orga** [comment]
> @stgmviknesh
> So I tried disabling _Preferences > Highlighting > Mark All > Match whole word only_
> And it looks this way:
> ![image](https://user-images.githubusercontent.com/9664141/128397936-f99ad711-244a-4b28-90dd-b33c923ed796.png)
> 
> Can you try the same?

This has fixed the issue for me as well. 

If I activate the setting "Match whole word only" again - why isn't it doing anything when a mark/highlight text like this:
![image](https://user-images.githubusercontent.com/85934624/130790899-1a8a9efa-6e2c-4c21-811f-2eb75fdcb8b9.png)
I would expect the highlighted word to be highlighted, but instead nothing happens. This feels wrong 😄

**Comment by gumida** [comment]
> @stgmviknesh
> So I tried disabling _Preferences > Highlighting > Mark All > Match whole word only_
> And it looks this way:
> ![image](https://user-images.githubusercontent.com/9664141/128397936-f99ad711-244a-4b28-90dd-b33c923ed796.png)
> 
> Can you try the same?

This has fixed the issue for me as well. 

If I activate the setting "Match whole word only" again - why isn't it doing anything when a mark/highlight text like this:
![image](https://user-images.githubusercontent.com/5578303/130791669-4fa5454e-0e83-4d30-9f7d-38a709af3649.png)
I would expect the highlighted word to be highlighted, but instead nothing happens. This feels wrong 😄

**Comment by ArkadiuszMichalski** [comment]
I suggest you update to the latest version, best would be 8.1.4. When you report a problem, fill  the Debug Information section, because no one will guess on which version you are working on.

**Comment by Juraj2** [comment]
Disabling the "Preferences > Highlighting > Mark All > Match whole word only" fixed the issue v8.1.4 32 bit.
